sdk/schemaregistry/azure-data-schemaregistry-avro/pom.xml

```diff
@@ -46,7 +46,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-data-schemaregistry</artifactId>
-      <version>1.0.0-beta.2</version> <!-- {x-version-update;com.azure:azure-data-schemaregistry;current} -->
+      <version>1.0.0-beta.3</version> <!-- {x-version-update;com.azure:azure-data-schemaregistry;current} -->
```

Comment:
top level version was not updated.. probably because version_client.txt was not updated